dev/acf-export.json

```diff
@@ -81,6 +81,66 @@
                 "multiple": 1,
                 "return_format": "id",
                 "ui": 1
+            },
+            {
+                "key": "field_60413825e0ada",
+                "label": "Header Story",
```

Comment:
Consider moving the new header story fields to a new, separate field group from EDU News Options (maybe named something like "EDU Header Story Options") to help separate the main site news feed options and header story options on the EDU News Feed options page.